### PR TITLE
BULK VALIDATION alt text, etc.

### PR DESCRIPTION
**Pull request by mSarts**

### CODE REVIEW

File: aspnet/web-forms/overview/deployment/advanced-enterprise-web-deployment/troubleshooting-the-packaging-process.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ If you're using a custom project file to build your projects, you can include th
 
 [!code-xml[Main](troubleshooting-the-packaging-process/samples/sample3.xml)]
 
-If you're using a Team Foundation Server (TFS) build definition to build your projects, you can supply a value for the **EnablePackageProcessLoggingAndAssert** property in the **MSBuild Arguments** row:![](troubleshooting-the-packaging-process/_static/image1.png)
+If you're using a Team Foundation Server (TFS) build definition to build your projects, you can supply a value for the **EnablePackageProcessLoggingAndAssert** property in the **MSBuild Arguments** row:![If you're using a Team Foundation Server (TFS) build definition to build your projects, you can supply a value for the EnablePackageProcessLoggingAndAssert property in the MSBuild Arguments.](troubleshooting-the-packaging-process/_static/image1.png)
```
Comment (rflynn27) [inline aspnet/web-forms/overview/deployment/advanced-enterprise-web-deployment/troubleshooting-the-packaging-process.md:56]: Same here

File: aspnet/web-forms/overview/deployment/configuring-team-foundation-server-for-web-deployment/deploying-a-specific-build.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ Until now, the topics in this tutorial set have focused on how to build, package
 
 Consider the continuous integration (CI) scenario described in the previous topic, [Creating a Build Definition That Supports Deployment](creating-a-build-definition-that-supports-deployment.md). You've created a build definition in Team Foundation Server (TFS) 2010. Every time a developer checks code into TFS, Team Build will build your code, create web packages and database scripts as part of the build process, run any unit tests, and deploy your resources to a test environment. Depending on the retention policy you configured when you created the build definition, TFS will retain a certain number of previous builds.
 
-![](deploying-a-specific-build/_static/image1.png)
+![Depending on the retention policy you configured when you created the build definition, TFS will retain a certain number of previous builds.](deploying-a-specific-build/_static/image1.png)
```
Comment (rflynn27) [inline aspnet/web-forms/overview/deployment/configuring-team-foundation-server-for-web-deployment/deploying-a-specific-build.md:30]: T F S

File: aspnet/web-forms/overview/deployment/configuring-team-foundation-server-for-web-deployment/deploying-a-specific-build.md [hunk 3]
```diff
@@ -62,28 +62,28 @@ In this case, you don't want the build definition to actually build anything&#x2
...
 1. In Visual Studio 2010, in the **Team Explorer** window, expand your team project node, right-click **Builds**, and then click **New Build Definition**.
 
-    ![](deploying-a-specific-build/_static/image2.png)
+    ![In Visual Studio 2010, in the Team Explorer window, expand your team project node, right-click Builds, and then click New Build Definition](deploying-a-specific-build/_static/image2.png)
 2. On the **General** tab, give the build definition a name (for example, **DeployToStaging**) and an optional description.
 3. On the **Trigger** tab, select **Manual – Check-ins do not trigger a new build**.
 4. On the **Build Defaults** tab, in the **Copy build output to the following drop folder** box, type the Universal Naming Convention (UNC) path of your drop folder (for example, **\\TFSBUILD\Drops**).
 
-    ![](deploying-a-specific-build/_static/image3.png)
+    ![On the Build Defaults tab, in the Copy build output to the following drop folder box, type the Universal Naming Convention (UNC) path of your drop folder (for example, \\TFSBUILD\Drops).](deploying-a-specific-build/_static/image3.png)
```
Comment (rflynn27) [inline aspnet/web-forms/overview/deployment/configuring-team-foundation-server-for-web-deployment/deploying-a-specific-build.md:70]: U N C

File: aspnet/web-forms/overview/deployment/deploying-web-applications-in-enterprise-scenarios/application-lifecycle-management-from-development-to-production.md [hunk 4]
```diff
@@ -182,7 +182,7 @@ When a build has been approved in the staging environment, the Fabrikam, Inc. te
 
 The production environment is in an Internet-facing perimeter network. This is isolated from the internal network that contains the build server. The production environment administrator, Lisa Andrews, must manually copy the web deployment packages from the build server and import them into IIS on the primary production web server.
 
-![](application-lifecycle-management-from-development-to-production/_static/image6.png)
+![The production environment administrator must manually copy the web deployment packages from the build server and import them into IIS on the primary production web server.](application-lifecycle-management-from-development-to-production/_static/image6.png)
```
Comment (rflynn27) [inline aspnet/web-forms/overview/deployment/deploying-web-applications-in-enterprise-scenarios/application-lifecycle-management-from-development-to-production.md:185]: I I S

File: aspnet/web-forms/overview/deployment/web-deployment-in-the-enterprise/manually-installing-web-packages.md [hunk 5]
```diff
@@ -79,18 +79,18 @@ For more information on configuring an IIS web server to support web deployment
 
 1. In IIS Manager, in the **Connections** pane, right-click your IIS website, point to **Deploy**, and then click **Import Application**.
 
-    ![](manually-installing-web-packages/_static/image3.png)
+    ![In IIS Manager, in the Connections pane, right-click your IIS website, point to Deploy, and then click Import Application.](manually-installing-web-packages/_static/image3.png)
```
Comment (rflynn27) [inline aspnet/web-forms/overview/deployment/web-deployment-in-the-enterprise/manually-installing-web-packages.md:82]: I I S

File: aspnet/web-forms/overview/deployment/web-deployment-in-the-enterprise/manually-installing-web-packages.md [hunk 6]
```diff
@@ -103,10 +103,10 @@ For more information on configuring an IIS web server to support web deployment
 6. Click **Next**.
 7. If this is not the first time you've deployed the application to this website, you'll be prompted to specify whether you want to delete all existing content prior to installation. Choose the option that's appropriate for your requirements, and then click **Next**.
 
-    ![](manually-installing-web-packages/_static/image6.png)
+    ![Choose the option that's appropriate for your requirements, and then click Next.](manually-installing-web-packages/_static/image6.png)
 8. When IIS has finished installing the package, click **Finish**.
 
-    ![](manually-installing-web-packages/_static/image7.png)
+    ![When IIS has finished installing the package, click Finish.](manually-installing-web-packages/_static/image7.png)
```
Comment (rflynn27) [inline aspnet/web-forms/overview/deployment/web-deployment-in-the-enterprise/manually-installing-web-packages.md:109]: I I S

**Comment by Rick-Anderson** [comment]
@mSarts  fix conflicts

**Comment by mSarts** [comment]
@Rick-Anderson Merge conflicts addressed. Ready for your review.

**Pull request merged**